test/ssc_test/cmod_json_comparison_test.h

```diff
@@ -58,11 +58,15 @@ class JSONComparisonTest : public ::testing::Test {
 
 public:
     void Test(const std::string& compute_module, const std::string& file_inputs, const std::string &file_outputs, const std::vector<std::string> &compare_number_variables, const std::vector<std::string> &compare_array_variables) {
+        char solar_resource_path[256];
```

Comment:
Looks like the error tolerance on lines 92 and 101 is too tight for the linux vs mac differences. Is a blanked increase to 0.01 ok, or do we need to do this on a percentage basis?

Reply:
tolerance updated for macOS tests. All tests now pass
![image](https://user-images.githubusercontent.com/6498311/191491099-12feb517-aa76-4a08-8291-8a1d6b4b77b5.png)